physionet-django/console/views.py

```diff
@@ -818,17 +872,25 @@ def manage_published_project(request, project_slug, version):
     - Deprecate files
     - Create GCP bucket and send files
     """
+    from console.utility import get_bucket_name_and_prefix, check_s3_bucket_with_prefix_exists, has_aws_credentials, missing_S3_open_data_info
```

Comment:
imports should be at the top of the module.

Reply:
@tompollard I didn't add the import at the top of the module in some cases because I was encountering import errors like this:

ImportError: cannot import name 'get_file_forms' from partially initialized module 'project.views' (most likely due to a circular import)